src/hooks/useHtmlPaste/index.ts

```diff
@@ -146,7 +149,7 @@ const useHtmlPaste: UseHtmlPaste = (textInputRef, preHtmlPasteCallback, removeLi
             unsubscribeBlur = navigation.addListener('blur', () => document.removeEventListener('paste', handlePaste));
         }
 
-        document.addEventListener('paste', handlePaste);
+        document.addEventListener('paste', handlePaste, true);
```

Comment:
The `removeEventListener` call parameters should match the `addEventListener` otherwise it won't work https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener

Reply:
Changed